src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj

```diff
@@ -155,7 +155,7 @@
     <Compile Include="Utility\NullableNuGetVersionConverter.cs" />
     <Compile Include="Utility\LogEvents.cs" />
     <Compile Include="Utility\ILoggerFactoryExtensions.cs" />
-    <Compile Include="Utility\SafeExceptionConverter.cs" />
+    <Compile Include="Utility\SafeExceptionConverter.cs" /
```

Comment:
accident?

Reply:
Yup, just fixed it.

I resolved a merge conflict using GitHub UI and it appears to have been the culprit